application/values.yaml

```diff
@@ -187,6 +187,9 @@ deployment:
     #       values:
     #       - ssd
 
+  # Number of ReplicaSet versions to retain
+  revisionHistoryLimit: 2
```

Comment:
I understand that #245 stipulates a default value of 2 here for performance gains in avoiding keeping old `ReplicaSets` around. However, one of the reasons to use Kubernetes is to have an easier time rolling changes back. Could this value affect the possibility of rollbacks through Helm, or, if any brave soul is mixing it into production, `kubectl rollback`?

This is of course not enough to warrant a _major_ version bump, but in my opinion a warning about this new default value and its repercussions might be warranted in the project's README.

Reply:
What do you think about adding this warning in the Changelog instead?